cli/Valet/Site.php

```diff
@@ -759,6 +768,24 @@ function proxyDelete($url)
         info('Valet will no longer proxy [https://'.$url.'].');
     }
 
+    /**
+     * Create the given nginx host.
+     *
+     * @param  string  $url
+     * @param  string  $siteConf  pregenerated Nginx config file contents
+     * @return void
+     */
+    function put($url, $siteConf)
+    {
+        $this->unsecure($url);
```

Comment:
So, because this has `unsecure` in it, I think we may need to rename it. It may have to be something like `putInsecurely` or something? I know that's awkward, but the idea that this thing that's simply "putting" it *also* insecures it seems a bit unexpected. That said, I'm not *super* familiar with this class; @drbyte do you have other suggestions?